### ISSUE TITLE
Use of --resume in training and culling newly downloaded images for object quality

### ISSUE THREAD

**Issue by Philliec459** [dusty-nv/jetson-inference#660]
@dusty-nv , your full inclusion of pytorch-ssd into this jetson-inference repository is great. I really like the ability to download additional open_images with labels as required. Thank you.

I have a question on --resume. I initially trained using 30 epochs and then used --resume to train for an additional 30 epochs. On --resume I used the path of the last epoch from the first training session. Having watched the epoch Loss over the initial training session, it did not appear that --resume started where I left off.  This is the command that I used:

python3 train_ssd.py --model-dir=models/marine --batch-size=4 --num-epochs=30 --resume=/home/craig/src/jetson-inference/python/training/detection/ssd/models/marine/mb1-ssd-Epoch-29-Loss-2.176339123307205.pth

I was wondering if I missed anything in the command line that I used?

Also, I used all of the downloaded images in my first attempt at this and the results were fair. I then went though all of the newly downloaded images and removed images that were from a wrong perspective for my use or images that did not appear to have well defined objects for my purposes. The second attempt at training showed a much lower loss with what I thought was better object detection. However, did I loose anything statistically by high-grading my dataset?

**Comment by Philliec459** [comment]
I guess this in not important
